lib/src/ui/pluto_left_frozen_rows.dart

```diff
@@ -71,3 +71,108 @@ class PlutoLeftFrozenRowsState
     );
   }
 }
+
+class FrozenRow extends StatefulWidget {
+  final int rowIdx;
+
+  final PlutoRow row;
+
+  final List<PlutoColumn> columns;
+
+  final PlutoGridStateManager stateManager;
+
+  final bool visibilityLayout;
+
+  const FrozenRow({
+    required this.rowIdx,
+    required this.row,
+    required this.columns,
+    required this.stateManager,
+    this.visibilityLayout = false,
+    super.key,
+  });
+
+  @override
+  State<FrozenRow> createState() => _FrozenRowState();
+}
+
+class _FrozenRowState extends State<FrozenRow> {
+  final LayerLink _layerLink = LayerLink();
+  OverlayEntry? _leftFloatingEntry;
+
+  _showOverlay() {
+    if (_leftFloatingEntry != null) {
+      _leftFloatingEntry?.remove();
+      _leftFloatingEntry = null;
+    }
+
+    Widget? floatingWidget =
+        widget.stateManager.rowLeftFloatingWidgetCallback?.call(
+      PlutoRowContext(
+        row: widget.row,
+        rowIdx: widget.rowIdx,
+        stateManager: widget.stateManager,
+      ),
+    );
+
+    if (floatingWidget == null) return;
+
+    _leftFloatingEntry = OverlayEntry(
+      builder: (context) => Positioned(
+        left: 0,
+        child: CompositedTransformFollower(
```

Comment:
what is this `CompositedTransformFollower` for?

Reply:
Just inserting the floating widget on the app Overlay would not be enought. I had to make this floating widget follow the respective row on user scrolling. In order to a widget follow the position of another widget, there is the `CompositedTransformFollower` and `CompositedTransformTarget`. The follower follows the position of the target through the `LayerLink` link.

Reply:
I couldn't find a way to put the floating widget over the row and translate its position to outside the grid without the grid border hides the widget. So, I had to put it on the app Overlay.